app/build.gradle

```diff
@@ -3,11 +3,12 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'androidx.navigation.safeargs.kotlin'
+apply plugin: 'com.google.gms.google-services'
```

Comment:
shouldn't this plugin be applied as last at the very bottom?

Reply:
I was under the same impression but it seems like it might no longer be necessary: https://firebase.google.com/docs/android/setup#console